### PR TITLE
opensuse-toolbox-tumbleweed: Add more packages

### PR DESCRIPTION
**Pull request by TheEvilSkeleton**
This supposedly addresses https://github.com/toolbx-images/images/pull/31#issuecomment-1274970094.

I changed the email to fedoraproject.org and also separated the commands. It's easier and faster to test when some commands are already cached.

### CODE REVIEW

**Comment by travier** [comment]
The idea behind having a single RUN command ist to reduce the number of layers and the overall size by cleaning up repo metadata after the update. This Containerfile does not do it yet however.

**Review by travier — APPROVED** [review]
LGTM. Not merging directly to let you optionally address my comment above.

**Comment by TheEvilSkeleton** [comment]
Does this look good?

**Comment by TheEvilSkeleton** [comment]
Does this look good?

File: opensuse/tumbleweed/Containerfile [hunk 1]
```diff
@@ -6,13 +6,16 @@ LABEL com.github.containers.toolbox="true" \
...
 
 # Install extra packages
 COPY extra-packages /
 RUN zypper update -y && \
     cat /extra-packages | xargs zypper install -y
 RUN rm /extra-packages
 
-# Enable password less sudo
+# Clean up cache
+RUN zypper clean
```
Comment (travier) [inline opensuse/tumbleweed/Containerfile:18]: The idea is to run this one in the same RUN command as the one above to cleanup the cache and not include it in the image layer at all.

File: opensuse/tumbleweed/Containerfile [hunk 2]
```diff
@@ -6,13 +6,16 @@ LABEL com.github.containers.toolbox="true" \
       version="$VERSION" \
       usage="This image is meant to be used with the toolbox command" \
       summary="Base image for creating openSUSE toolbox containers" \
-      maintainer="Hari Rana <theevilskeleton@riseup.net>"
+      maintainer="Hari Rana <theevilskeleton@fedoraproject.org>"
 
 # Install extra packages
 COPY extra-packages /
 RUN zypper update -y && \
     cat /extra-packages | xargs zypper install -y
```
Comment (travier) [inline opensuse/tumbleweed/Containerfile:14]:
```suggestion
    cat /extra-packages | xargs zypper install -y && \
    zypper clean    
```

File: opensuse/tumbleweed/Containerfile (hunk 1 above)
Comment (TheEvilSkeleton) [reply opensuse/tumbleweed/Containerfile:19]: What is "this one"? GitHub's review system is unclear for me.

Comment (travier) [reply opensuse/tumbleweed/Containerfile:19]: This command line 18

Comment (travier) [reply opensuse/tumbleweed/Containerfile:19]: So now that it's in the RUN command above, we don't need it here anymore

File: opensuse/tumbleweed/Containerfile [hunk 3]
```diff
@@ -6,13 +6,17 @@ LABEL com.github.containers.toolbox="true" \
       version="$VERSION" \
       usage="This image is meant to be used with the toolbox command" \
       summary="Base image for creating openSUSE toolbox containers" \
-      maintainer="Hari Rana <theevilskeleton@riseup.net>"
+      maintainer="Hari Rana <theevilskeleton@fedoraproject.org>"
 
 # Install extra packages
```
Comment (travier) [inline opensuse/tumbleweed/Containerfile:11]:
```suggestion
# Update packages, install extra packages and clean up cache 
```

File: opensuse/tumbleweed/Containerfile [hunk 4]
```diff
@@ -6,13 +6,17 @@ LABEL com.github.containers.toolbox="true" \
...
 COPY extra-packages /
 RUN zypper update -y && \
-    cat /extra-packages | xargs zypper install -y
+    cat /extra-packages | xargs zypper install -y && \
+    zypper clean    
 RUN rm /extra-packages
 
-# Enable password less sudo
+# Clean up cache
```
Comment (travier) [inline opensuse/tumbleweed/Containerfile:18]:
```suggestion
```

File: opensuse/tumbleweed/Containerfile [hunk 5]
```diff
@@ -6,13 +6,17 @@ LABEL com.github.containers.toolbox="true" \
...
+RUN zypper clean
```
Comment (travier) [inline opensuse/tumbleweed/Containerfile:19]:
```suggestion
```

File: opensuse/tumbleweed/Containerfile [hunk 6]
```diff
@@ -6,13 +6,15 @@ LABEL com.github.containers.toolbox="true" \
...
+# Update packages, install extra packages and clean up cache 
 COPY extra-packages /
 RUN zypper update -y && \
-    cat /extra-packages | xargs zypper install -y
+    cat /extra-packages | xargs zypper install -y && \
+    zypper clean    
 RUN rm /extra-packages
 
-# Enable password less sudo
+
```
Comment (travier) [inline opensuse/tumbleweed/Containerfile:18]:
```suggestion
```

**Comment by travier** [comment]
I've added my suggestions. I'll let you squash commits if you agree with them.

**Comment by TheEvilSkeleton** [comment]
Should be good to go!

**Review by travier — APPROVED** [review]

**Pull request merged**